### ISSUE TITLE
SilenceService crashes Android7 devices

### ISSUE THREAD

**Issue by pfent** [TCA-Team/TumCampusApp#418]
Probably this issue on StackOverflow:
http://stackoverflow.com/questions/39151453/in-android-7-api-level-24-my-app-is-not-allowed-to-mute-phone-set-ringer-mode

**Comment by deepeshp** [comment]
hey pfent ! We are doing the exact same thing in our code too so issue seems to be the same, i have asked the permission in code specifically for android 7.0 for this bug but not able to push my local branch.<some permissions issue.let me know how can i proceed forward in this.

**Comment by pfent** [comment]
You can usually open a pull request, if you want to contribute code to our app. That way, we can review the code and have a basic sanity check.

You probably want to:
```bash
git remote add deepeshp git@github.com:deepeshp/TumCampusApp.git
git push -u deepeshp master
```
You then can [open a pull request](https://github.com/TCA-Team/TumCampusApp/compare), that shows your changes and we'll proceed to merge these

**Comment by kordianbruck** [comment]
Last fix #392 worked till Android 6, but the way we are checking it was really a hack. Heres another collection of suggestions on how to fix it in Android 7
http://stackoverflow.com/questions/39151453/in-android-7-api-level-24-my-app-is-not-allowed-to-mute-phone-set-ringer-mode
